src/lib/server.ts

```diff
@@ -354,8 +354,11 @@ export class ZwavejsServer extends EventEmitter {
 }
 
 export class EventEmitterLogTransport extends Transport {
+  private messageSymbol: Symbol;
+
   public constructor(private emitter: EventEmitter, private logger: Logger) {
     super({ format: createDefaultTransportFormat(false, false) });
+    this.messageSymbol = Symbol.for("message");
```

Comment:
Are you actually using this?

Reply:
thanks I think I lost something on rebase